packages/jest-snapshot/src/utils.js

```diff
@@ -178,13 +178,32 @@ export const saveSnapshotFile = (
   );
 };
 
+const deepMergeArray = (target, source) => {
```

Comment:
Inspired by the [`deepmerge` documentation](https://www.npmjs.com/package/deepmerge).

I looked at replacing `util.deepMerge()` with that npm package since it's already used indirectly by `jest-website`, but it looks like we had to implement the array merging logic manually either way.